test/unittests/t_options.cc

```diff
@@ -0,0 +1,90 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+
+#include "gtest/gtest.h"
+
+#include "../../cvmfs/options.h"
+#include "../../cvmfs/util.h"
+
+using namespace std;  // NOLINT
+
+namespace options {
+
+class T_Options : public ::testing::Test {
+ protected:
+  virtual void SetUp() {
+    string filename;
```

Comment:
`filename` doesn't seem necessary since there is already the `config_file` field in the class.